### ISSUE TITLE
同样的网络，Mac连不上，但苹果手机能连上

### ISSUE THREAD

**Issue by mikolaje** [qiuyuzhou/ShadowsocksX-NG#1384]
上个月都还能连，但不知道为最近突然连不上。同样的地址，苹果手机连接没有任何问题。

![image](https://user-images.githubusercontent.com/11773334/129720596-073c1396-059a-43e7-a947-bcf989957630.png)

![image](https://user-images.githubusercontent.com/11773334/129720659-29ac3a37-98a9-409f-b57b-4ead9dbac3a3.png)

检查电脑本地端口没有1080，不知道是否正常现象。

![image](https://user-images.githubusercontent.com/11773334/129720684-4f37ebe2-ef8b-4cdf-9f23-947dd27e78e9.png)

求教。

The version is `Version 1.4.4-R8 (1)`

**Comment by yikerman** [comment]
@mikolaje I don’t think this is the UI of ShadowsocksX-NG.

**Comment by mikolaje** [comment]
> @mikolaje I don’t think this is the UI of ShadowsocksX-NG.
yes it is
![image](https://user-images.githubusercontent.com/11773334/129732278-9b08dcb3-b305-4857-a611-eb11bde05660.png)

**Comment by yikerman** [comment]
No it isn’t.

It says R8, so maybe it’s ShadowsocksX-NG-R, which supports ShadowsocksR.

Most of the time it’s forked from SSX-NG, and maybe the author just forgot to modify the issue report URL.

@mikolaje

**Comment by mikolaje** [comment]
> No it isn’t.
> 
> It says R8, so maybe it’s ShadowsocksX-NG-R, which supports ShadowsocksR.

OK, maybe it's one of the stream of the ShadowsocksX-NG.
Anyway, do you have any suggestion for this kind of problem? I spent my all day but cannot find any solution. So appreciated for any help of you.

**Comment by yikerman** [comment]
@mikolaje

You can checkout the log file, located in `Library/Logs/ss-local.log` or such kind of names. Since you are using ShadowsocksR, I can’t promise. Maybe ss-local crashed.

**Comment by yikerman** [comment]
@mikolaje BTW ShadowsocksX-NG-R is located at https://github.com/qinyuhang/ShadowsocksX-NG-R/tree/develop. But I won’t consider it as actively maintained.

**Comment by mikolaje** [comment]
> @mikolaje BTW ShadowsocksX-NG-R is located at https://github.com/qinyuhang/ShadowsocksX-NG-R/tree/develop. But I won’t consider it as actively maintained.

It seemed working after downloading your latest version of ShadowsocksX-NG :)  Thanks you so much!

BTW, how can I add a subscribed link?  I can't find such feature in ShadowsocksX-NG.

Thanks~

**Comment by yikerman** [comment]
@mikolaje As you can see, ShadowsocksX-NG-R has been abandoned for a long time and I don’t think there IS such thing.

**Comment by mikolaje** [comment]
> @mikolaje As you can see, ShadowsocksX-NG-R has been abandoned for a long time and I don’t think there IS such thing.

I mean ShadowsocksX-NG.  I'm using ShadowsocksX-NG now.

**Comment by yikerman** [comment]
No idea. 

What does ‘my latest version’ mean?

@mikolaje

**Comment by mikolaje** [comment]
> No idea.
> 
> What does ‘my latest version’ mean?
> 
> @mikolaje
I mean this version
![image](https://user-images.githubusercontent.com/11773334/129738520-3f62ed6d-744f-4671-99f1-a74938a0b1af.png)

**Comment by xiaoyu2006** [comment]
@mikolaje Oh I see. Let me check it out.

**Comment by yikerman** [comment]
@mikolaje I don’t think this feature is implemented. Leave a feature request about it. (SIP008)

Or maybe you want to leave it in my continued repo at https://gitHub.com/ssx-ng/ShadowsocksX-NG.

**Comment by mikolaje** [comment]
> @mikolaje I don’t think this feature is implemented. Leave a feature request about it. (SIP008)
> 
> Or maybe you want to leave it in my continued repo at https://gitHub.com/ssx-ng/ShadowsocksX-NG.

OK, check it later. thanks.

**Comment by DCSuperhero** [comment]
The same question, Did you solve it?

**Comment by ruoshi93** [comment]
The same question here, but I think that's because I use the IP instead of the domain name. After I changed it, it works.

**Comment by YihangFu** [comment]
It happens while you are using macOS Monterey.
